spec/features/user_adds_new_spot_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+
+# As an authenticated user
+# I want to create a new installation "spot"
+# So that I can share interesting art that I found
+
+# Acceptance Criteria
+# [] I should see an “Add Art Spot” link in the top bar, visible from every page.’
+# [] I should see an “Add New Art Spot” callout link in the body of the page.
+# [] I should add a new art spot and save it to the database.
+
+feature 'authenticated user adds new spot' do
+  before(:each) do
+    user = FactoryBot.create(:user)
+    visit new_user_session_path
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
+  end
```

Comment:
You can skip this by using the `login_in` method. 
https://github.com/plataformatec/devise/wiki/How-To:-Test-with-Capybara
Or configure your own `sing_in` helper method for tests

Reply:
But I will leave that up to you